CI fails with dev dependencies: ImportError: cannot import name 'TRANSFORMERS_CACHE' from 'transformers.utils.hub'

CI fails with dev dependencies: https://github.com/huggingface/trl/actions/runs/19759515173/job/56617955722?pr=4596
> ImportError: cannot import name 'TRANSFORMERS_CACHE' from 'transformers.utils.hub'
```python
==================================== ERRORS ====================================
___________ ERROR collecting tests/experimental/test_bco_trainer.py ____________
ImportError while importing test module '/__w/trl/trl/tests/experimental/test_bco_trainer.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/__w/_tool/Python/3.12.12/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/experimental/test_bco_trainer.py:27: in <module>
    from ..testing_utils import TrlTestCase, require_no_wandb, require_peft, require_sklearn
tests/testing_utils.py:34: in <module>
    from trl.experimental.judges import BaseBinaryJudge, BasePairwiseJudge
trl/experimental/judges/__init__.py:15: in <module>
    from .judges import (
trl/experimental/judges/judges.py:28: in <module>
    import llm_blender
.venv/lib/python3.12/site-packages/llm_blender/__init__.py:1: in <module>
    from llm_blender.blender.blender import Blender
.venv/lib/python3.12/site-packages/llm_blender/blender/blender.py:11: in <module>
    from .blender_utils import (
.venv/lib/python3.12/site-packages/llm_blender/blender/blender_utils.py:22: in <module>
    from transformers.utils.hub import TRANSFORMERS_CACHE
E   ImportError: cannot import name 'TRANSFORMERS_CACHE' from 'transformers.utils.hub' (/__w/trl/trl/.venv/lib/python3.12/site-packages/transformers/utils/hub.py)

...

======================= 12 warnings, 34 errors in 43.39s =======================
```

--- a/trl/experimental/judges/judges.py
+++ b/trl/experimental/judges/judges.py
@@ -58,17 +58,15 @@ def _ensure_llm_blender_importable() -> None:
     """
     Pre-import shim to work around a known `llm-blender` issue.
 
-    As of `llm-blender` v0.0.2 (see upstream issue:
-    https://github.com/yuchenlin/LLM-Blender/issues/33), importing
-    `llm_blender` may fail on `transformers` >= 5.0.0.dev0 because it
-    unconditionally accesses `transformers.utils.hub.TRANSFORMERS_CACHE`.
+    As of `llm-blender` v0.0.2 (see upstream issue: https://github.com/yuchenlin/LLM-Blender/issues/33), importing
+    `llm_blender` may fail on `transformers` >= 5.0.0.dev0 because it unconditionally accesses
+    `transformers.utils.hub.TRANSFORMERS_CACHE`.
 
-    We set this attribute to a dummy value before importing
-    `llm_blender` so that the import succeeds. This helper is intentionally
-    a no-op on older `transformers` versions.
+    We set this attribute to a dummy value before importing `llm_blender` so that the import succeeds. This helper is
+    intentionally a no-op on older `transformers` versions.
 
-    This shim can be removed once the upstream issue is fixed and the minimum
-    required `llm-blender` version includes that fix.
+    This shim can be removed once the upstream issue is fixed and the minimum required `llm-blender` version includes
+    that fix.
     """
     import transformers.utils.hub
 

